app/controllers/index.js

```diff
@@ -25,7 +25,7 @@ export default class IndexController extends Controller {
       this.toggleUserStateModal();
     }
     try {
-      await fetch(`${BASE_URL}/users/status/self`, {
+      await fetch(`${BASE_URL}/users/status/self?userStatusFlag=true`, {
```

Comment:
What if we want to change it to false ?
What's the use of a feature flag when we can't change it without changing the code ??

Reply:
https://firebase.google.com/docs/remote-config

Reply:
> What if we want to change it to false ? Can we utilize the flag better without changing the code ??

if we pass the flag as false it will work as usual accepting the active and idle state. which will no longer be accepted in the new flow

Reply:
> https://firebase.google.com/docs/remote-config

Sure we can go ahead with this once Ankush and Tejas agrees.